src/OmniSharp.Bootstrap/project.json

```diff
@@ -13,19 +13,21 @@
     "omnisharp.bootstrap": "run"
   },
   "frameworks": {
-    "dnx451": {
+    "net451": {
       "frameworkAssemblies": {
         "System.Runtime": ""
       }
     },
-    "dnxcore50": {
-      "imports": "portable-net45+win8",
+    "netstandard1.5": {
+      "imports": [
+        "dotnet5.6",
```

Comment:
The dnxcore50 import might hide the need for it so I don't know, but I would love to get rid of the dnxcore50 import first.